sormas-e2e-tests/src/test/java/org/sormas/e2etests/entities/services/api/EventApiService.java

```diff
@@ -61,6 +67,7 @@ public Event buildGeneratedEvent() {
         .srcType(SourceTypeValues.getRandomSourceTypeName())
         .eventInvestigationStatus("PENDING")
         .eventTitle(String.valueOf(System.currentTimeMillis()))
+        .eventDesc(faker.chuckNorris().fact())
```

Comment:
applies to all event creation scenarios?

Reply:
Yes